python/lsst/ts/fiberspectrograph/csc.py

```diff
@@ -249,9 +249,9 @@ async def do_expose(self, data):
             await self.fault(code=20, report=msg)
             raise salobj.ExpectedError(msg)
 
-        await self.save_data(spec_data)
+        await self.save_data(spec_data, data.groupId)
```

Comment:
As I mentioned in the electrometer PR, you might want to ensure this change is backwards compatible, for the time being.